packages/core/js-client/src/network.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type Node = { peerId: string; multiaddr: string };
+
+export const stage: Node[] = [
```

Comment:
I do not know how to fix it, but this is a very likely point of inconsistency among the mapping across fluence repos. 

Could we reuse the @nahsi mapping book here? https://github.com/fluencelabs/fluence-network-environment/blob/main/src/index.ts

Otherwise, it seems we should remember the sources we ought to go through and update the address when we decide to update

Reply:
the idea is to maybe deprecate fluence-network-environment package completely so there is less stuff to think about